templates/_layouts/_header.html

```diff
@@ -25,9 +25,10 @@
       </ul>
       {% if path.startswith('/docs') %}
       <form class="p-search-box" action="/docs/search">
-        <input type="search" id="search-docs" class="p-search-box__input" name="q" {% if query %}value="{{ query }}"{% endif %} placeholder="Search the docs" title="Search the documentation" autocomplete="on" required />
-        <button type="reset" id="search-docs-reset" class="p-search-box__reset u-no-margin--right" name="close"><i class="p-icon--close"></i></button>
-        <button type="submit" class="p-search-box__button" name="submit"><i class="p-icon--search"></i></button>
+        <label class="u-off-screen" for="search-docs">Search the docs</label>
+        <input type="search" id="search-docs" class="p-search-box__input" name="q" {% if query %}value="{{ query }}"{% endif %} placeholder="Search the docs" title="Search the documentation"  autocomplete="on" required />
+        <button type="reset" id="search-docs-reset" class="p-search-box__reset u-no-margin--right" name="close"><i class="p-icon--close"></i><span class="u-off-screen">Clear input</span></button>
```

Comment:
You can add the text inside the icon as the content is `u-off-screen` by default.

Reply:
Oh good shout, didn't know that